roles/edpm_ovs_dpdk/tasks/configure.yml

```diff
@@ -208,6 +208,30 @@
       ansible.builtin.set_fact:
         edpm_ovs_external_ids: "{{ {} | combine(ovn_bridge_datapath_type) }}"
 
+- name: Set shared memory pool config
+  when: edpm_ovs_dpdk_shared_mem_pool|string
+  block:
+    - name: Apply shared memory pool config
+      ansible.builtin.set_fact:
+        dpdk_shared_mem_pool:
+          shared-mempool-config: "{{ edpm_ovs_dpdk_shared_mem_pool }}"
```

Comment:
are these set_fact tasks actually needed, or it possible to use the role variable directly in the combine filter?

`
"{{ edpm_ovs_other_config | combine({'shared-mempool-config': dpdk_shared_mem_pool}) }}"
`